api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TXTBCohortQueries.java

```diff
@@ -494,7 +494,7 @@ public CohortDefinition getPatientsStartTBTreatment() {
             + "    o.concept_id = ${tbDrugTreatmentStartDate} AND "
             + "    e.encounter_datetime BETWEEN :startDate AND :endDate AND "
```

Comment:
If this method is nolonger applicable, you can consider deleting it

Reply:
public CohortDefinition getPatientsStartTBTreatment()

Reply:
Removed